src/components/nodes-table/SmartView.vue

```diff
@@ -505,10 +505,16 @@ export default {
 				icon = mdiPowerPlug
 				description = 'Main power source'
 			} else {
-				description = Array.isArray(node.batteryLevels)
-					? 'All battery levels: ' +
-					  node.batteryLevels.map((v) => `${v}%`).join(',')
-					: 'Unknown battery level'
+				description =
+					node.batteryLevels !== null &&
+					typeof node.batteryLevels === 'object' &&
+					!Array.isArray(node.batteryLevels)
```

Comment:
This check is useless because they could never be an array now

Reply:
Makes sense. Will update PR in a moment.